db/structure.sql

```diff
@@ -1985,36 +2018,37 @@ CREATE SEQUENCE public.messages_id_seq
 
 
 --
--- Name: messages_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: -
+-- Name: nameservers_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: -
 --
 
-ALTER SEQUENCE public.messages_id_seq OWNED BY public.messages.id;
+ALTER SEQUENCE public.nameservers_id_seq OWNED BY public.nameservers.id;
 
 
 --
--- Name: nameservers; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+-- Name: notifications; Type: TABLE; Schema: public; Owner: -; Tablespace: 
 --
 
-CREATE TABLE public.nameservers (
+CREATE TABLE public.notifications (
     id integer NOT NULL,
-    hostname character varying NOT NULL,
-    ipv4 character varying[] DEFAULT '{}'::character varying[],
+    registrar_id integer NOT NULL,
+    text character varying NOT NULL,
+    attached_obj_type character varying,
+    attached_obj_id integer,
+    read boolean NOT NULL,
```

Comment:
Would setting default to false have some adverse effects here? We could get rid of the `Notification#set_defaults` method.

Reply:
Considered it, but decided in favor of `set_defaults` to be able to know the value before inserting to a database. Thought it would simplify things a bit.